src/hello-world.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+
+int HelloWorld (int argc, char** argv)
```

Comment:
This isn't the API you declared in the header file.

Reply:
I will go ahead and update that now.